qa/suites/rados/encoder/1-tasks.yaml

```diff
@@ -1,57 +1,42 @@
 tasks:
-- print: "**** install version -2 (quincy) ****"
+- print: "**** install version -2 (reef) ****"
```

Comment:
Oh, I bet we should be changing this test every single version bump up. How about adding a new bullet to the `doc/dev/release-checklists.rst` to not forget?

Reply:
Yes! will add it to the doc!